libs/cdk/utils/drag-and-drop/dnd-list/dnd-list.directive.ts

```diff
@@ -122,7 +126,7 @@ export class DndListDirective<T> implements AfterContentInit, OnDestroy {
     private readonly _refresh$ = new Subject<void>();
 
     /** @hidden */
-    private readonly _onDestroy$ = new Subject<void>();
+    private readonly _onDestroy$ = inject(DestroyRef);
```

Comment:
When I moved away from DestroyService to using DestroyRef, I used name `_destroyRef` for class properties, which I believe works better, because it is not neither event, nor the observable, simply a ref